apis/analysis/v1alpha1/recommendation_types.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RecommendationTargetType defines the type of analysis target
+type RecommendationTargetType string
+
+const (
+	// RecommendationTargetWorkload defines the k8s workload type
+	RecommendationTargetWorkload RecommendationTargetType = "workload"
```

Comment:
```suggestion
	RecommendationTargetWorkload RecommendationTargetType = "Workload"
```

Reply:
we use camel format with lowercase just like other json format definition.

Reply:
In K8s type convention, a type shall use PascalCase, e.g. https://github.com/kubernetes/api/blob/master/autoscaling/v2/types.go#L232